UserAdminBundle/Tests/Functional/Controller/FormControllersSecurityTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace OpenOrchestra\UserAdminBundle\Tests\Functional\Controller;
+
+use OpenOrchestra\BackofficeBundle\Tests\Functional\Controller\AbstractControllerTest;
+
+/**
+ * Class FormControllersSecurityTest
+ */
+class FormControllersSecurityTest extends AbstractControllerTest
+{
+    protected $username = 'userLog';
```

Comment:
when this PR https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1152 is merge, use this use : userNoAccess